src/qutip_qip/operations/__init__.py

```diff
@@ -31,7 +31,7 @@
 #    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 from .gates import (
-    rx, ry, rz, x_gate, y_gate, z_gate, s_gate, t_gate,
+    Gate, rx, ry, rz, x_gate, y_gate, z_gate, s_gate, t_gate,
```

Comment:
As we already declared `__all__` in `qutip_qip/operations/__init___.py`, we are safe to use `from .gates import *`. Only what declared in `__all__` in `gates.py` will be imported.

What you did is perfectly correct, just this could be a minor simplification for your future work on `gates.py`